pkg/apis/v1beta1/labels.go

```diff
@@ -89,6 +89,7 @@ var (
 	ResourceNVIDIAGPU          v1.ResourceName = "nvidia.com/gpu"
 	ResourceAMDGPU             v1.ResourceName = "amd.com/gpu"
 	ResourceAWSNeuron          v1.ResourceName = "aws.amazon.com/neuron"
+	ResourceXilinxAccelerator  v1.ResourceName = "xilinx.com/fpga-xilinx_u30_gen3x4_base_1-0"
```

Comment:
I'm not sure how rare it is to get this instance type, but to validate the resource name in general we should try to get an E2E test setup for this like we have with the habana instance